src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -162,16 +459,37 @@
 		"sourcePath" : "log.sagebase.org/teamMemberSnapshots/",
 		"tableDescriptor": {
 			"name": "teammembersnapshots",
-			"columns": {
-				"change_timestamp": "timestamp",
-				"snapshot_timestamp": "timestamp",
-				"team_id": "bigint",
-				"member_id": "bigint",
-				"is_admin": "boolean"
+			"description" : "This table contain snapshots of team-members. Snapshots are taken when members are added or deleted from a team. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.",
```

Comment:
Same question as above for distinguishing the difference between snapshots created via node creation/update/deletion vs the periodic snapshot taking.

Reply:
Same explanation as above